.github/workflows/cmake.yml

```diff
@@ -0,0 +1,134 @@
+name: CMake
+
+on: [push]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
```

Comment:
Don't build `Release` on push. Build and test `Debug` on push, build `RelWithDebInfo` on release.

Reply:
Back when I was writing conduit, I was using a beta version of GCC 10 (for some reason that was the default that shipped with ubuntu 20.04). It had a couple bugs that only appeared when the code was compiled with Release, and I had to work around them. I started compiling with Release so that the unit tests would check to ensure those bugs didn't reappear.

Reply:
Then we can set the release action to be runnable manually and before merge, but it shouldn't be run on every push. I don't know if I have enough container hours 😂

Reply:
Github actions is free for public repositories!

![image](https://user-images.githubusercontent.com/36810712/128107795-1fcb6d10-7eda-4ff2-b970-5fa930a77f9a.png)